pkgs/top-level/python-packages.nix

```
@@ -6338,6 +6338,8 @@ in {
 
   python_statsd = callPackage ../development/python-modules/python_statsd { };
 
+  skein = disabledIf isPy27 (callPackage ../development/python-modules/skein { });
```

buildPythonPackage can handle this in the expression
```suggestion
  skein = callPackage ../development/python-modules/skein { };
```